src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
+  pageTitle="Buttons"
+>
+  <sky-docs-demo-page-summary>
+    The button classes create buttons to trigger actions from within an interface. For information about styles and implementation, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/buttons-links">buttons and links guidelines</a>.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Button classes"
+    >
+      <table class="sky-docs-table">
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-primary">Primary</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The primary button triggers the most important or most frequently used action on a page or the progressive terminal action on a modal (such as <strong>Save</strong> or <strong>Done</strong>). Pages and modals should only include one primary button to ensure that it draws user attention. For information about how to use primary actions, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/page-layouts#primary-action-guidelines">primary action guidelines</a>.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-primary&quot;&gt;Primary&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-default">Secondary</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Secondary buttons trigger less important or less frequently used actions. Since secondary buttons are less important than primary buttons, SKY UX reduces the visual weight to make them less attention-drawing than the primary buttons.
```

Comment:
I wonder if we really need the opening clause of the second sentence.

Reply:
I agree and removed it.